setup.py

```diff
@@ -1,11 +1,14 @@
 import setuptools
```

Comment:
You mentioned on Slack that only "some dev cases" require a `setup.py`. Do we depend on those? If not, how about we just delete this?

Reply:
yeah, I think we should be fine deleting the setup.py